ansible/verify_config.py

```diff
@@ -0,0 +1,768 @@
+import argparse
+import csv
+import glob
+import logging
+import os
+import subprocess
+from abc import ABC, abstractmethod
+from collections import deque
+from enum import Enum
+from functools import lru_cache
+
+import yaml
+from yaml.parser import ParserError
+
+"""
+Script Usage Guide
+
+This script is designed to validate our configurations as part of our documentation process.
+
+NOTE: For now this script only supports yaml format
+
+General Usage:
+
+To verify the entire project’s configuration:
```

Comment:
Can we mention that running `python3 verify_config.py` will use the default testbed file name `testbed.yaml` and default vm file name `veos`, please?